### ISSUE TITLE
Getting token not working for me

### ISSUE THREAD

**Issue by pjacobr** [BastianPoe/owlet_api#2]
I am trying to get the CLI and code to work but it isn't working using my owlet email and password. Do you know if the API has changed since January? I decompiled the app and it looks like the api might have moved over to owlets own servers now.

stringBuilder.append("https://ss-registration");
      stringBuilder.append(param1ad.a());
      stringBuilder.append(".owletdata.com/cleanup/");

    stringBuilder.append("https://ayla-sso");
      stringBuilder.append(param1ad.a());
      stringBuilder.append(".owletdata.com/mini/");

**Comment by hesto2** [comment]
This is not working for me either.

**Comment by stillsober** [comment]
Yeah, as far as I can tell nobody has a working piece of code for this. Owlet redesigned their app and moved their iot service. It may not be as easy as before to hack it. I'd love to hear some new info if anybody has some.

**Comment by BastianPoe** [comment]
> I currently do no have my owlet Handy and hence was not able to try this out.

But I will have it again soon and then give it a try.

**Comment by hesto2** [comment]
Just stumbled across this, haven't tested it yet but just decompiled my app and can verify the values are solid

**Comment by hesto2** [comment]
Decompiled the app and pulled out some of the new values for a project I was working on. Not super familiar with python otherwise I would've tried to just make a PR to this lib. Here is the npm package/repo for the small client I pulled together with their new authentication system:
https://github.com/hesto2/owlet-client

**Comment by isupremedyou** [comment]
I was unable to get a token using the CLI, @BastianPoe did you ever get this figured out? I so wish they just had an export option for the data

**Comment by BastianPoe** [comment]
I have started working on changes quite some time ago, but never got to completing them. I have now added them to the branch https://github.com/BastianPoe/owlet_api/tree/feature/support-new-auth-mechanism. Feel free to try them out and contribute.

**Comment by jamesbiederbeck** [comment]
@hesto2 I tried to use your client, but I'm a typescript noob. Thought I could just pipe your sample code into nodejs, and I had a Bad Time. 

Can I hassle you for clearer usage documentation, or a more friendly package?

**Comment by hesto2** [comment]
@jamesbiederbeck Ah man, I must have it configured wrong on my end. You shouldn't need to have typescript in your project to use the client. I'll fix that this weekend and then get back to you.

**Comment by freekeys** [comment]
Hey @BastianPoe - is it possible to merge the new auth mechanism into the owlet-py pip install? Or is there still work to do? What is still needed? Thanks!

**Comment by andrew-lozoya** [comment]
https://github.com/BastianPoe/owlet_api/tree/feature/support-new-auth-mechanism 
 
 doesnt work either :(

**Comment by mys007** [comment]
The implementation at https://github.com/jlamendo/ha-sensor.owlet/blob/main/custom_components/owlet/sensor.py currently works, at least in my case (EU region).

**Comment by andrew-lozoya** [comment]
@mys007 I ended up building my own, https://github.com/andrew-lozoya/owletsock-newrelic
